tools/pythonpkg/tests/fast/udf/test_null_filtering.py

```diff
@@ -0,0 +1,72 @@
+import duckdb
+import pytest
+
+pd = pytest.importorskip("pandas")
+pa = pytest.importorskip("pyarrow")
+from typing import Union
+import pyarrow.compute as pc
+
+from duckdb.typing import *
+
+
+class TestUDFNullFiltering(object):
+    @pytest.mark.parametrize(
+        'table_data',
+        [
+            ['12', None, '42'],
+            [None, '42', None],
+            ['12', None, None],
+            [None, None, '42'],
+            [None, None, None],
+        ],
+    )
+    def test_null_filtering_native(self, duckdb_cursor, table_data):
+        null_count = sum([1 for x in table_data if not x])
+        row_count = len(table_data)
+
+        df = pd.DataFrame({'a': table_data})
+        duckdb_cursor.execute("create table tbl as select * FROM df")
+
+        def my_native_func(x):
```

Comment:
Can we add some tests with functions that receive multiple input parameters as well?